v1/website/static/js/code-blocks-buttons.js

```diff
@@ -44,4 +44,25 @@ window.addEventListener('load', function() {
       textEl.textContent = 'Copy';
     }, 2000);
   });
+
+  // add event listener for all tab
+  document.querySelectorAll('.nav-link').forEach(function(el) {
```

Comment:
We will need to make sure that this is installed as part of the Docusaurus install process -- e.g., `docusaurus-init`